README.md

```diff
@@ -10,31 +11,75 @@ This chart will deploy the following on the target OpenShift cluster:
+- Logged in to a Red Hat OpenShift Container Platform (version 4.13+) cluster as a cluster administrator.
 - [OpenShift CLI (oc)](https://docs.openshift.com/container-platform/4.13/cli_reference/openshift_cli/getting-started-cli.html) is installed.
 - [Operator Lifecycle Manager (OLM)](https://olm.operatorframework.io/docs/getting-started/) has been installed in your cluster.
 - Your cluster has a [default storage class](https://docs.openshift.com/container-platform/4.13/storage/container_storage_interface/persistent-storage-csi-sc-manage.html) provisioned.
 - [Helm](https://helm.sh/docs/intro/install/) v3.9+ is installed.
 - [PostgreSQL](https://www.postgresql.org/) database is available with credentials to manage the tablespace (optional).
-  - A [reference implementation](#postgresql-deployment-reference-implementation) is provided for your convenience.
-- A Github API Token - to import items into the catalog, there is a need for GITHUB_TOKEN with the permissions as detailed [here](https://backstage.io/docs/integrations/github/locations/). For classic token, include the following permissions: repo (all), admin:org (read:org) and user (read:user, user:email).
-- `ArgoCD/OpenShift GitOps` operator is installed and one instance of `ArgoCD` exists in a given namespace (later referenced by `ARGOCD_NAMESPACE` env var)
+  - A [reference implementation](https://github.com/parodos-dev/orchestrator-helm-chart#deploying-postgresql-reference-implementation) is provided for your convenience.
+- A GitHub API Token - to import items into the catalog, ensure you have a GITHUB_TOKEN with the necessary permissions as detailed [here](https://backstage.io/docs/integrations/github/locations/). For classic token, include the following permissions:
+  - repo (all)
+  - admin:org (read:org)
+  - user (read:user, user:email)
+
+## Installation
+
+### Quick installation
+1. Follow instruction [here](https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/GitOps.md)
+2. Run the following command to set up environment variables:
+    ```console
+    ./hack/setenv.sh
+    ```
+  - Accept all default values
+  - Provide GH Token (created as describe in prerequisite section)
+3. Source the environment variables by running
+    ```console
+    source .env
+    ```
+4. Install the orchestrator chart using one of the following options:
+   1. **Option 1: Install the chart with SonataFlow services in ephemeral mode for evaluation purposes**
+      ```console
+      helm install orchestrator orchestrator --set orchestrator.devmode=true \
+          --set rhdhOperator.github.token=$GITHUB_TOKEN \
+          --set rhdhOperator.k8s.clusterToken=$K8S_CLUSTER_TOKEN --set rhdhOperator.k8s.clusterUrl=$K8S_CLUSTER_URL \
+          --set argocd.namespace=$ARGOCD_NAMESPACE --set argocd.url=$ARGOCD_URL --set argocd.username=$ARGOCD_USERNAME \
+          --set argocd.password=$ARGOCD_PASSWORD --set argocd.enabled=true --set tekton.enabled=true
```

Comment:
I am conflicted here: do we consider the Tekton and ArgoCD mandatories? We may want to run only in dev mode without any other thing

Maybe also mention that the Tekton and ArgoCD are not mandatories (if they are not) but needed to have the full experience. But as it's for dev purposes, we can immagine the dev knows what it needs to test

Reply:
There was another section before `Quick installation without GitOps` describing this exact setup